### PR TITLE
Spelling Errors

### PR DESCRIPTION
**Pull request by Grauenwolf**
Spelling errors (and conventions) for parameters, comments, and local variables.

### CODE REVIEW

File: src/GraphQL/GraphQL.csproj [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="GraphQL-Parser" Version="3.0.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.0" />
```
Comment (SimonCropp) [inline src/GraphQL/GraphQL.csproj]: adding a new nuget as part of a "spelling fix" PR seems a bit heavy handed?

**Comment by Grauenwolf** [comment]
It's the same static code analysis feature that's part of Visual Studio. They just moved it to a package for .net core.

**Comment by Grauenwolf** [comment]
It's not a runtime dependency and it will help to prevent future spelling errors.

**Comment by SimonCropp** [comment]
@Grauenwolf yep i understand what it is. but the description of the PR doesnt really capture the scope of the change. I know i have seem, and experienced myself, many problem with fxcop over the years. and hence prefer to avoid it these days.

**Comment by SimonCropp** [comment]
@joemcbride given this now conflicts. would you like a new PR?

**Comment by joemcbride** [comment]
Closing as a new PR with fixes has been made.

**Pull request closed without merging**